### ISSUE TITLE
Trian Code

### ISSUE THREAD

**Issue by 1764758458** [hustvl/EVF-SAM#43]
Could you please explain what the attention_masks, offset, and label_list variables are in the sam2 training code, and how can I generate them?

**Comment by CoderZhangYx** [comment]
You don't need to manually generate them, they are available in our dataset code which is open-sourced.

For explanation, `attention_masks` is to mask the padding tokens of batched input texts for mm_extractor; `offset` has been explained in former issues #9 ; `label_list` is for passing the original image shape to the post-process functions.
